modules/lib.lua

```diff
@@ -395,6 +395,23 @@ else
         return GetLabelText(GetMakeNameFromVehicleModel(GetEntityModel(vehicle)))
     end
 
+    ---Returns the mod type name of the given vehicle.
+    ---@param vehicle integer
+    ---@param modType integer
+    ---@param modIndex integer
+    ---@return string
```

Comment:
Are there external documentation links that might be helpful here? I'm not sure what a mod type nor mod index are

Reply:
Mod type & mod index are very self explanatory,
type can be the transmission, turbo, exhaust etc
and index is.. the index in that type